addons/website/views/res_config_settings_views.xml

```diff
@@ -29,6 +33,7 @@
                             <div class="content-group">
                                 <div class="row mt8">
                                     <label class="col-lg-3" string="Domain" for="website_domain"/>
+                                    <a href="https://www.odoo.com/documentation/17.0/applications/websites/website/configuration/domain_names.html#domain-name-website-map" title="Documentation" class="o_doc_link" target="_blank"></a>
```

Comment:
Auto-review:
It is not well displayed
<img width="583" alt="image" src="https://github.com/odoo/odoo/assets/46810843/97096e3f-87ff-491e-bf43-007ea0838d0f">